src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    switch (action.type) {
+      default: {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

wrong place for default